src/onInit/cleanData.js

```diff
@@ -0,0 +1,24 @@
+import { set } from 'd3';
+
+export default function cleanData() {
+    //Remove missing and non-numeric data.
+    const preclean = this.raw_data,
+        clean = this.raw_data.filter(d => /^-?[0-9.]+$/.test(d[this.config.value_col])),
+        nPreclean = preclean.length,
+        nClean = clean.length,
+        nRemoved = nPreclean - nClean;
+
+    //Warn user of removed records.
```

Comment:
I think we should make this warning visible at some point. Make it a footnote or something instead of showing it just in the console. Created #79 for later.

Reply:
Not a bad idea, perhaps a red exclamation point with a tooltip.